lib/p2p/Pool.ts

```diff
@@ -193,6 +198,15 @@ class Pool extends EventEmitter {
     }
   }
 
+  public sendToPeer = (pubKey:string, packet:Packet): Error|undefined => {
+    const peer = this.peers.get(pubKey);
+    if (!peer) {
+      return new Error('Peer Not found');
```

Comment:
Don't like it. I consider throw a bad thing and try to avoid using it.

Reply:
But this would diverge with the error handling approach in the rest of the codebase, as well as internal NodeJS code from which errors are thrown and must be caught.